### DISCUSSION TITLE
Multi Select?

### DISCUSSION THREAD

**Discussion by chobo2** [vikramlearning/blazorbootstrap#441] [Ideas]
Hi

Is there going to be a multi selected added in the near future? 

Something like this?

https://react-select.com/home

**Comment by gvreddy04** [comment]
> Hi
> 
> 
> 
> Is there going to be a multi selected added in the near future? 
> 
> 
> 
> Something like this?
> 
> 
> 
> https://react-select.com/home

Please create an issue for this. We will add in the future releases.

Please add the reference links in the issue as well.
